readyplayerme/rpm-unity-sdk-photon-support
# [SDK-544] Feature/minor updates

﻿




## [SDK-544](https://ready-player-me.atlassian.net/browse/SDK-544)

## Description

-  update menu item priority for better separation
- updated avatar model name top prevent naming conflict

![Screenshot 2023-10-10 082817](https://github.com/readyplayerme/rpm-unity-sdk-netcode-support/assets/7085672/873fb049-5ffc-4798-8778-90b0d6fc3704)







## How to Test

-  Open the network-packaging-test branch on Unity SDK or import both Netcode and Photon packages into a project and check that there is no conflicts (Previously there was as both had the same model asset file name)



## Checklist



[SDK-544]: https://ready-player-me.atlassian.net/browse/SDK-544?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
